### ISSUE TITLE
Use namespaces, not classes

### ISSUE THREAD

**Issue by thekid** [xp-framework/rfc#317]
# Scope of Change

This RFC suggests updating coding standards to use namespaces instead of classes, inspired by Go language (and a bit by C#).
# Rationale
# Functionality
## Current

``` php
use util\{Date, DateUtil};
use util\cmd\Console;

$span= DateUtil::timespanBetween(new Date($argv[1]), Date::now());
Console::writeLine($span);
```
## New

``` php
use util, util\cmd;

$span= util\DateUtil::timespanBetween(new util\Date($argv[1]), util\Date::now());
cmd\Console::writeLine($span);
```
# Security considerations
# Speed impact
# Dependencies
# Related documents

**Comment by mikey179** [comment]
I was suggesting that for when a bunch of classes from a certain namespace gets used, I'd certainly not recommend that for single classes. The use statements should give a first impression of what will be used in the class below. A single, two or three classes from a namespace should be explicitly listed. In the example we looked at yesterday I'd only do such a namespace use for the rdbms classes. It makes sense there as the whole class was about working with the database.

**Comment by thekid** [comment]
Solved via #318
